crates/flow-client/control-plane-api.graphql

```diff
@@ -415,8 +433,16 @@ type LiveSpecRefEdge {
 	cursor: String!
 }
 
+input LiveSpecsBy @oneOf {
+	prefixAndType: ByPrefixAndType
+	names: [Name!]
+}
+
 """
-Represents an optimistic lock failure when trying to update live specs.
+Represents an optimistic lock failure when trying to update live specs. This
+typically means that an `expectPubId` was not matched after a successful
+build, meaning that another publication may have changed the spec while this
```

Comment:
nit: perhaps:

Represents an optimistic lock failure when trying to update a specification.
This typically means that an `expectPubId` was not matched, implying that another
publication has applied a conflicting update to this specification since it was last fetched.


I'm trying to capture that this can also happen because you're working with an outdated view and need to `flowctl catalog pull-specs`.